core/cloudflow-akka/src/main/scala/cloudflow/akkastream/AkkaStreamlet.scala

```diff
@@ -55,9 +58,20 @@ abstract class AkkaStreamlet extends Streamlet[AkkaStreamletContext] {
 
       if (activateCluster && localMode) {
         val clusterConfig = ConfigFactory.parseResourcesAnySyntax("akka-cluster-local.conf")
-        val fullConfig    = clusterConfig.withFallback(updatedStreamletDefinition.config)
-
-        val system  = ActorSystem(streamletDefinition.streamletRef, ConfigFactory.load(fullConfig))
+        // Add persistence configuration
+        val clusterConfigDB =
+          AkkaPersistenceConfigurator.getPersistenceParamsLocal(updatedStreamletDefinition.config, streamletDefinition.streamletRef) match {
+            case Some(dBConfiguration) =>
+              AkkaPersistenceConfigurator.addPersistenceConfiguration(clusterConfig, dBConfiguration)
+            case _ => clusterConfig
+          }
+        // Add user defined
+        val fullConfig = clusterConfigDB.withFallback(updatedStreamletDefinition.config)
+        // Create Akka system
+        val system = ActorSystem(streamletDefinition.streamletRef, fullConfig)
+        // Populate db tables
+        TableCreator.createTables(system)
```

Comment:
This should not always happen, should be a user choice, I think.

Reply:
Take a look at implementation, its configurable. Its also in documentation